### DISCUSSION TITLE
Cannot use created role to view app

### DISCUSSION THREAD

**Discussion by joeytzy** [Budibase/budibase#11389] [Help] [answered]
![image](https://github.com/Budibase/budibase/assets/44018390/fe4d4ee2-a3c9-4500-aaaa-6485576a387a)
Hi, so i created a new role called EDITOR which inherits from the POWER role as shown above. Basically this role is similar to POWER but has less permission.

![image](https://github.com/Budibase/budibase/assets/44018390/4896830b-b2f4-470e-9aeb-09374a998483)
When i want to preview as the role as shown above, there is no option for me to select that role.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @joeytzy,

Currently we only allow for the default created roles to be used to preview your application. At the moment you would have to login as the user with the specific role and view the app that way for testing purposes. 

As I do believe that you should be able to include custom roles in this dropdown when developing. I've create a feature request here; https://github.com/Budibase/budibase/issues/11402

I hope this helps.

**Reply by joeytzy** [reply]
thanks for clarifying it.
